apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -23,7 +23,13 @@ let schoolData = {
 
 $(document).ready(() => {
   const schoolInfoMountPoint = document.getElementById("school-info-inputs");
-  renderSchoolInfo();
+  init();
+
+  function init() {
```

Comment:
What's the advantage of `init()` here?

Reply:
once i added a second step on page load (setUserType and renderSchoolInfo), i figured it would be cleaner to just make a method